conf/abi.xml

```diff
@@ -204,6 +204,13 @@
       <field name="throttle"  type="int16_t">Throttle input in pprz_t [0;9600] or [-9600;9600] (for vertical speed control for instance)</field>
     </message>
 
+    <message name="OPTICAL_FLOW_INT32" id="29">
```

Comment:
There is already an OPTICAL_FLOW message, and beside some values that could be ignored by the new module, the only difference is the type of `flow_` fields.
If it is not possible to fit in a `int16_t` here, why not make it to `int32_t` for everyone ? Actually, `uint32_t` is strange, no negative values ? Is is really in deg/s ?
Making to message for the same topic is annoying because it means that data consumers will need to bind to both of them.

Reply:
Thank you for the comment, unfortunately there is zero documentation on the sensor so am not fully sure that the unit is that one. I will conduct more tests and try to implement it in the already existing message.

Reply:
If you can't find the units, it is not a big deal, it is probably sensor dependent anyway. The most important to me is merging the two OPTICAL_FLOW messages.